projects/demo-integrations/cypress/tests/kit/multi-select/multi-select.spec.ts

```diff
@@ -23,4 +23,25 @@ describe('MultiSelect', () => {
             .wait(DEFAULT_TIMEOUT_BEFORE_ACTION)
             .matchImageSnapshot(`01-not-overflow-by-tags`);
     });
+
+    it('checking that the arrow icon is rotated when enabled tuiTextfieldCleaner', () => {
+        cy.goToDemoPage(
+            'components/multi-select/API?tuiMode=null&tuiTextfieldCleaner=true',
```

Comment:
If I've not mistaken, `/API` can cause problem in launching this test from our proprietary repository

Reply:
<img width="1373" alt="image" src="https://user-images.githubusercontent.com/12021443/154266799-063d795c-b6e2-4b7d-8b58-5a237056a53c.png">

but other tests also use the API path